samples/apps/copilot-chat-app/webapi/Hubs/ChatHub.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.SignalR;
+
+namespace SemanticKernel.Service.SignalR.Hubs;
+
+public class ChatHub : Hub
+{
+    public async Task SendMessageAsync(string user, string message)
+    {
+        await this.Clients.All.SendAsync("SendMessage", user, message);
+        Console.WriteLine("Log SendMessage: {0}", message);
+    }
+
+    public async Task SendMessageToAllUsersExceptSelfAsync(string ClientSideCallBackName, string message)
+    {
+        await this.Clients.AllExcept(this.Context.ConnectionId).SendAsync(ClientSideCallBackName, message);
+        Console.WriteLine("Log SendMessageToAllUsersExceptSelfAsync: {0}", message);
```

Comment:
I'm not sure we want to log the user's messages because of PII and privacy concerns in general.